lib/src/main/java/com/auth0/android/lock/internal/json/Strategy.java

```diff
@@ -0,0 +1,33 @@
+package com.auth0.android.lock.internal.json;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import static com.auth0.android.util.CheckHelper.checkArgument;
+
+class Strategy {
```

Comment:
@hzalaz It was only used on the JSON parsing stage for better readability. Now I've moved all the logic in the `ApplicationDeserializer` class.